OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,433 @@
+Building an OUD image with WebLogic Image Tool
+===========================================
+
+## Contents
+
+1. [Introduction](#1-introduction)
+2. [Prerequisites](#2-prerequisites)
+3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
+4. [Download the required packages/installers](#4-download-the-required-packagesinstallers-patches)
+5. [Required build files](#5-required-build-files)
+6. [Additional build commands](#6-additional-build-commands)
+7. [Steps to Create OUD image](#7-steps-to-create-oud-image)
+8. [Sample Dockerfile generated with imagetool](#8-sample-dockerfile-generated-with-imagetool)
+
+# 1. Introduction
+
+This README describes the steps involved in building an OUD image with the WebLogic Image Tool.
+
+# 2. Prerequisites
+
+The following prerequisites are necessary before building OUD Docker images with Image Tool:
```

Comment:
Same as before. Please review all use of the word "Docker" and remove it as much as possible, or replace it with phrases like "container image"

Reply:
Taken care of this at lot of the instances. Remaining instances are to be updated through future PR.